### PR TITLE
Update assemblyscript.md

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com

Closes https://github.com/fermyon/developer/issues/1197

### CODE REVIEW

**Comment by tpmccallum** [comment]
https://github.com/AssemblyScript/wasi-shim

File: content/wasm-languages/assemblyscript.md [hunk 1]
```diff
@@ -53,32 +53,32 @@ This section provides a basic example of building AssemblyScript from source.
...
 
-```console
+```bash
 $ npm init
 # Answer questions
-$ npm install --save @assemblyscript/loader
+```
 
-added 1 package, and audited 2 packages in 986ms
+Install the TS compiler:
```
Comment (dicej) [inline content/wasm-languages/assemblyscript.md:61]:
```suggestion
Install the ActionScript compiler:
```

File: content/wasm-languages/assemblyscript.md [hunk 2]
```diff
@@ -53,32 +53,32 @@ This section provides a basic example of building AssemblyScript from source.
...
 Now use `npx` to scaffold out your new project:
 
 ```console
 $ npx asinit .
-Version: 0.19.23
+Version: 0.27.24
 
 This command will make sure that the following files exist in the project
-directory '/Users/technosophos/Code/AssemblyScript/hello':
+directory '/Users/Fermyon/typescript-example':
```
Comment (dicej) [inline content/wasm-languages/assemblyscript.md:81]: Where does this path come from?  It seems unlikely that the user is going to have a `/Users/Fermyon` directory on their filesystem.  And Linux users won't have a `/Users` directory at all.

File: content/wasm-languages/assemblyscript.md [hunk 3]
```diff
@@ -101,144 +101,190 @@ directory '/Users/technosophos/Code/AssemblyScript/hello':
...
   ./tests/index.js
-  Example test to check that your module is indeed working.
+  Stater test to check that the module is functioning.
+
+  ./index.html
+  Starter HTML file that loads the module in a browser.
 
 The command will try to update existing files to match the correct settings
-for this instance of the compiler in '/Users/technosophos/Code/AssemblyScript/hello/node_modules/assemblyscript'.
+for this instance of the compiler in '/Users/Fermyon/typescript-example/node_modules/assemblyscript'.
```
Comment (dicej) [inline content/wasm-languages/assemblyscript.md:111]: Ditto the previous comment.  Can we make the path more system-agnostic?

File: content/wasm-languages/assemblyscript.md (hunk 1 above)
Comment (itowlson) [reply content/wasm-languages/assemblyscript.md:61]: ActionScript or AssemblyScript?

Comment (dicej) [reply content/wasm-languages/assemblyscript.md:61]: Fixed, thanks.  I was having Flash/SWF flashbacks, I guess.

Comment (tpmccallum) [reply content/wasm-languages/assemblyscript.md]: Thanks @dicej and @itowlson 
The latest commit has these changes in.

File: content/wasm-languages/assemblyscript.md (hunk 2 above)
Comment (tpmccallum) [reply content/wasm-languages/assemblyscript.md]: Thank you! The latest commit has these changes in.
I ran it all again locally, works well.

File: content/wasm-languages/assemblyscript.md (hunk 3 above)
Comment (tpmccallum) [reply content/wasm-languages/assemblyscript.md]: Done, thanks.

File: content/wasm-languages/assemblyscript.md [hunk 4]
```diff
@@ -53,32 +53,34 @@ This section provides a basic example of building AssemblyScript from source.
 
 Set up the project like this:
 
-```console
+```bash
+$ mkdir typescript-example
```
Comment (itowlson) [inline content/wasm-languages/assemblyscript.md:57]: Actually the question about the "typescript" naming belongs here I guess - sorry!

**Review by itowlson — APPROVED** [review]
LGTM!

**Pull request merged**